iotilecore/iotile/core/utilities/async_tools/operation_manager.py

```diff
@@ -316,8 +402,45 @@ def wait_for(self, timeout=None, **kwargs):
                 if message_val in next_level:
                     to_check.append(next_level[message_val])
 
+        if len(processors) > 0 and wait:
+            results = await asyncio.gather(*processors, return_exceptions=True)
+            for proc, result in zip(processors, results):
+                if isinstance(result, Exception) and not isinstance(result, asyncio.CancelledError):
+                    self._logger.error("Error running processor %s: %s", proc, result)
+
         return not ignored
 
+    async def process_all(self):
```

Comment:
If this is a public method, should it have a short docstring for consistency (or is that overkill)? Saw it in a few places, and it seems obvious enough, but we're slightly diverging from our style guide I think (perhaps the language could be softened... particularly if we start introducing type hints)

Reply:
I updated all of the docstrings to make them compliant with our style guide.  Thanks for checking.